pilot/pkg/bootstrap/server.go

```diff
@@ -88,6 +88,10 @@ var (
 		plugin.Mixer,
 		plugin.Envoyfilter,
 	}
+
+	// Default is enabled - can be set to "0" to restore previous behavior, in case of problems.
+	// Will be removed in 1.1 or 1.0.4 if we see no issues.
```

Comment:
Please move this to https://github.com/istio/istio/blob/master/pkg/features/pilot/pilot.go 
I don't want to cleanup undocumented environment variables once again

Reply:
There is no such thing in 1.0
When I merge - I'll not add the old code.